commands/log.js

```diff
@@ -1,87 +1,54 @@
 const readLastLines = require(`read-last-lines`);
-
-async function get_logs(server, size, sl, msg, internal_error) {
+async function get_logs(server, size, msg) {
     readLastLines.read(`/home/factorio/servers/eu-0${server}/console.log`, size) //D:\\programing\\lua\\factorio\\servers\\s${server}\\data\\console.log
-        .then((lines) => {
-            lines = lines.replace(/```/g, ',,,');
-            lines = lines.split('\n');
-            lines = lines.slice(sl-1, size+1);
-            lines = lines.join('\n');
-            lines = lines.match(/[\s\S]{1,1500}/g);
-
-            for (let i = 0; i < lines.length; i++) {
-                msg.channel.send(`\`\`\`${lines[i]}\`\`\``)
-                    .catch((err) => {internal_error(err); return});
-            }
-        }) // lines gets returned with last X (see above) lines and sent to same channel as command
+        .then((lines) => { let finalLines = lines.replace(/```/g, ',,,'); msg.channel.send(`\`\`\`${finalLines}\`\`\``, { split: true }) }) // lines gets returned with last X (see above) lines and sent to same channel as command
```

Comment:
```suggestion
        .then((lines) => { let finalLines = lines.replace(/```/g, ',,,'); msg.channel.send(`\`\`\`log \n${finalLines}\n\`\`\``, { split: true }) }) // lines gets returned with last X (see above) lines and sent to same channel as command
```
To get formatting correct this may work.

Reply:
o f i did not change this yet to the bigger messeges thing ill do that and then fix this too